service/src/java/org/apache/hive/service/cli/thrift/ThriftCLIService.java

```diff
@@ -530,10 +600,9 @@ public TCloseSessionResp CloseSession(TCloseSessionReq req) throws TException {
       SessionHandle sessionHandle = new SessionHandle(req.getSessionHandle());
       cliService.closeSession(sessionHandle);
       resp.setStatus(OK_STATUS);
-      ThriftCLIServerContext context =
-        (ThriftCLIServerContext)currentServerContext.get();
+      ThriftCLIServerContext context = (ThriftCLIServerContext) currentServerContext.get();
```

Comment:
Nit: formatting only?

Reply:
Well, you know me, I try to fix up what I can.  There is no need for this line break based on current checkstyle rules, but I can revert.